content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,175 @@ aliases:
+This new username/password combination should be used within the data source configurations to continue to provide read-only access to the underlying data.
+
+Ensure your dashboards are all functioning before upgrading.
+
+### Other data
 
-{{% note %}}
 The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
-{{% /note %}}
 
-1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/),
-   unpackage the InfluxDB binaries, and then place them in your `$PATH`.   
+### Secure by default
+
+Running InfluxDB with the setting
+
+```
+auth-enabled = false
+```
+
+is no longer supported in InfluxDB 2.0.
+By default authentication is required.
+After upgrading, authentication errors can occur if the username/password credentials within InfluxDB do not match what is currently being used by the various applications, agents, visualization tools, etc.
+Consider enabling authentication *before* upgrading to ensure that you have the appropriate credentials in place.
+
+### Available operating system, container, and platform support
+
+InfluxDB 2.0 is currently available for MacOS and Linux.
+Docker images, ARM builds, and Windows builds are not currently available.
+These are planned for subsequent releases.
+
+There is no plan for a 32-bit build of InfluxDB 2.0.
+
+## Perform the upgrade
+
+If you've considered the guidance above in [Before you begin](#before-you-begin-important-considerations) and are ready to proceed, follow these steps to upgrade your InfluxDB 1.x to InfluxDB 2.0.
+
+{{% warn %}}
+*After* stopping your InfluxDB 1.x process, make a back-up copy of all 1.x data before upgrading with `influx upgrade`.
+
+```sh
+cp -R .influxdb/ .influxdb_bak/
+```
+{{% /warn %}}
+
+1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/).
+   Unpackage the InfluxDB binaries and place them in your `$PATH`.
 2. Stop your running InfluxDB 1.x instance.
-3. If your 1.x configuration file is at the
-  [default location](/influxdb/v1.8/administration/config/#using-the-configuration-file), run:
+3. Use `influxd version` to ensure you are running InfluxDB 2.0 from the command line.
+   The `influxd upgrade` command is only available in InfluxDB 2.0.
+4. If your 1.x configuration file is at the [default location](/influxdb/v1.8/administration/config/#using-the-configuration-file), run:
 
     ```sh
     influxd upgrade
     ```
-     Otherwise, run:
+
+    If your 1.x configuration file is not at the default location, run:
+
     ```sh
     influxd upgrade --config-file <path to v1 config file>
     ```
 
-4. Follow the prompts to set up a new InfluxDB 2.0 instance.
+5. Follow the prompts to set up a new InfluxDB 2.0 instance.
+
+   ```
+   Welcome to InfluxDB 2.0 upgrade!
+   Please type your primary username: buzz
+
+   Please type your password:
+
+   Please type your password again:
+
+   Please type your primary organization name: fizz
+
+   Please type your primary bucket name: fizzbuzz
+
```

Comment:
maybe we can replace buzz, fizz, fizzbuzz w "your username"...and so on

(for a minority without context/reading literal, eg, ESL), may interpret bucket as some combo of org and username